openaddr/conform.py

```diff
@@ -1014,6 +1014,19 @@ def conform_smash_case(source_definition):
             if "field_to_remove" in conform[k]:
                 conform[k]["field_to_remove"] = conform[k]["field_to_remove"].lower()
 
+            if "functions" in conform[k]:
+                if type(conform[k]["functions"]) is list:
+                    for i in range(len(conform[k]["functions"])):
```

Comment:
This could be changed to `for function in conform[k]["functions"]:` and then you would be able to replace later instances of `conform[k]["functions"][i]` with `function` to reduce the appearance of complex array/hash logic.